frontend/src/i18n/locales/en-us.json

```diff
@@ -78,7 +78,10 @@
     "amount-exceeds-balance": "Amount exceeds wallet balance",
     "please-complete-form": "Please complete the form",
     "wallet-not-connected": "Wallet not connected",
-    "umbra-fee-exceeds-balance": "required for Umbra fee exceeds wallet balance"
+    "umbra-fee-exceeds-balance": "required for Umbra fee exceeds wallet balance",
+    "advanced-send-warning": "The receiver will have to enter their account's private key into this app to withdrawal the funds. Don't use this feature unless you know what you're doing.",
+    "learn-more": "Learn more",
+    "acknowledge-risks": "I acknowledge the risks"
```

Comment:
_(arbitrary comment location to start a thread)_

The token  warning PR and send button modal have a checkbox to acknowledge, but the pubkey modal just has a button with no checkbox. Personally I like the checkbox so we should add it to be consistent

Reply:
Adding the checkbox is fine with me.